variables.tf

```
@@ -216,6 +216,12 @@ variable "identity_ids" {
   default     = []
 }
 
+variable "ftps_state" {
+  description = "State of FTP/FTPS service for this Windows Function App."
+  type        = string
+  default     = "Disabled"
+}
```

Validate allowed values (according to [best practices](https://equinor.github.io/terraform-baseline/best-practices/variables-and-outputs/)):

```suggestion
  default     = "Disabled"
  
  validation {
    condition     = contains(["AllAllowed", "FtpsOnly", "Disabled"], var.ftps_state)
    error_message = "FTPS state must be \"AllAllowed\", \"FtpsOnly\" or \"Disabled\"."
  }
}
```